apps/next/src/pages/Settings/components/NetworkManagement/components/Confirmations/Confirmation.tsx

```diff
@@ -0,0 +1,79 @@
+import { Button, Stack, Typography, useTheme } from '@avalabs/k2-alpine';
+import { PageTopBar } from '@/components/PageTopBar';
+import { MdInfoOutline } from 'react-icons/md';
+
+type ConfirmationProps = {
+  onBack: () => void;
+  onSubmit: () => void;
+  title: string;
+  warningText: string;
+  primaryButtonText: string;
+  secondaryButtonText: string;
+};
+
+export const Confirmation = ({
+  onBack,
+  onSubmit,
+  title,
+  warningText,
+  primaryButtonText,
+  secondaryButtonText,
+}: ConfirmationProps) => {
+  const theme = useTheme();
+
+  return (
+    <Stack
+      height="100cqh"
+      width={1}
+      bgcolor="background.backdrop"
+      overflow="hidden"
+    >
```

Comment:
To keep it consistent with the rest of UI where we ask for confirmations, let's make this component a `SlideUpDialog`.

```tsx
<SlideUpDialog open={...} onClose={onBack}>
  <Page title={title} onBack={onBack} ...>
    {/* contents */}
  </Page>
</SlideUpDialog>
```

This will make it slide-in from the bottom of the screen, like this:

https://github.com/user-attachments/assets/1c8aee57-d2ee-4866-a772-8c5c2e0fdc03